moveit_experimental/package.xml

```diff
@@ -1,21 +1,15 @@
 <package>
   <name>moveit_experimental</name>
   <version>0.10.6</version>
-  <description>Experimental packages for moveit.</description>
-  <author email="isucan@google.com">Ioan Sucan</author>
-  <author email="robot.moveit@gmail.com">Sachin Chitta</author>
-  <author email="acorn.pooley@sri.com">Acorn Pooley</author>
-  <author email="jnicho@swri.org">Jorge Nicho</author>
-
-  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
-  <maintainer email="chitt@live.in">Chittaranjan Srinivas Swaminathan</maintainer>
```

Comment:
I believe that manifests without any `maintainer` tags are invalid and will result in catkin erroring out.